schemas/mod-source-record-manager/jobMonitoring.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Job Monitoring Schema",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "$ref": "../common/uuid.json"
+    },
+    "jobExecutionId": {
+      "description": "Corresponding jobExecution id",
+      "type": "string"
+    },
+    "lastEventTimestamp": {
+      "description": "Date and time when the last event occurred",
+      "type": "string",
+      "format": "date-time"
+    },
+    "notification": {
+      "description": "Notification sent to the consumer",
+      "type": "string"
```

Comment:
As I remember from spike this parameter should be `boolean` with the name `notification_sent`. It should indicate whether notification sent or not.

Reply:
Agree, fixed it